CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,13 @@
+# Code of Conduct
+
+This project has adopted the [OCaml Code of Conduct](https://github.com/ocaml/code-of-conduct/blob/main/CODE_OF_CONDUCT.md).
+
+# Enforcement
+
+This project follows the OCaml Code of Conduct [enforcement policy](https://github.com/ocaml/code-of-conduct/blob/main/CODE_OF_CONDUCT.md#enforcement).
+
+To report any violations, please contact:
```

Comment:
This doesn't look like the full list of maintainers: https://github.com/ocurrent/ocaml-dockerfile/blob/master/dockerfile.opam#L9-L15